### ISSUE TITLE
DeprecationWarning: "@coroutine" decorator is deprecated since Python 3.8, use "async def" instead

### ISSUE THREAD

**Issue by Theelgirl** [dano/aioprocessing#30]
While this isn't crucial yet, it may become necessary in the near future to replace instances of @coroutine with the modern ``async def``, like in managers.py:132-134
```python
    @asyncio.coroutine                            
    def __aexit__(self, *args, **kwargs):
       self.release()
```
into
```python
    async def __aexit__(self, *args, **kwargs): 
       self.release()
```

**Comment by dano** [comment]
@Theelgirl thanks for the heads up. I've been leaving that alone to avoid breaking backwards compatibility with older versions of Python 3.x, but it looks like it's being removed in 3.10, so I'll have to deal with it.

**Comment by davesteele** [comment]
3.10 pre-releases have begun.

**Comment by dano** [comment]
Fixed in the 1.1.0 release
